index.ts

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+const todo = require("./src/todo");
+const MyTodo = new todo.Todo();
+
+app.use(express.json()) // for parsing application/json
+app.use(express.urlencoded({ extended: true })) // for parsing application/x-www-form-urlencoded
+
+app.get('/', function (req, res) {
+    const listDes = [];
+    if (MyTodo.list.length !== 0) {
+        const len = MyTodo.list.length;
+        for (let i = 0; i < len; i++) {
+            listDes.push(MyTodo.list[i].description)
+        }
+        res.send(listDes + '\n');
+    } else {
+        res.status(404);
+        res.send('No items in list. \n');
+    }
+});
+
+app.post('/', function (req, res) {
+    const todoItem = MyTodo.addItem(req.body.Description, req.body.Date, req.body.IsDone);
```

Comment:
So what happens if one of the parameters is missing?  Mayhaps some validation is in order.  If a field is missing throw a 400 (Bad Request), otherwise add the item and return the JSON object.  Again use `res.json`.